src/sentry/search/events/fields.py

```diff
@@ -2870,8 +2855,6 @@ def __init__(
                     default_result_type="number",
                     private=True,
                 ),
-                # TODO: implement these
-                SnQLFunction("absolute_delta", snql_aggregate=self._resolve_unimplemented_function),
```

Comment:
lol did we really implement it above and forget to delete this?

Reply:
😃 coding is hard